src/appl/jiv.c

```diff
@@ -972,6 +976,8 @@ static void render()
 	if (cmdopts.verbose) {
 //		fprintf(stderr, "vtlx=%f, vtly=%f, vsx=%f, vsy=%f\n",
 //		  vtlx, vtly, gs.sx, gs.sy);
+		/* suppress -Wunused-but-set-variable */
+		(void)vtlx; (void)vtly;
```

Comment:
I recommend to split statements [Declare one name (only) per declaration](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-name-one)
```suggestion
		(void)vtlx;
		(void)vtly;
```

Reply:
I agree on this one. It makes it more readable to me. But I don't care much and can merge anyways.

Reply:
This isn't supposed to be "readable". This is a kludge to work around a compiler warning for code which is commented out for unknown reasons. Usually, I wouldn't write two statements on one line, but in this case I decided to do so because I didn't want to waste any screen space for this cr*p.

Reply:
Not sure whether this is a good reason to break with the style ;)